packages/button/src/ButtonContent/DefaultContent.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+
+import ButtonIcon from '../ButtonIcon/ButtonIcon';
+import { ButtonProps, Size, Variant } from '../types';
+
+import {
+  buttonContentSizeStyle,
+  buttonContentStyle,
+  darkModeRightGlyphStyles,
+  leftGlyphStyles,
+  rightGlyphStyles,
+} from './ButtonContent.styles';
+
+type ButtonContentProps = Omit<ButtonProps, 'as'>;
+
+const DefaultContent = ({
+  leftGlyph,
+  rightGlyph,
+  className,
+  children,
+  variant = Variant.Default,
```

Comment:
What are your thoughts on making these required in `ButtonContentProps` so that you don't have to set a default since the defaults are already set in `ButtonContent`?

Reply:
Also, maybe it should be `DefaultContentProps` to match the component name?